css-color-4/Overview.bs

```diff
@@ -4147,7 +4152,8 @@ Predefined Color Spaces</h2>
 
 		</dl>
 
-<h3 id="predefined-a98-rgb">Predefined a98-rgb space</h3>
+<h3 id="predefined-a98-rgb">
+The Predefined opRGB Color Space: the ''a98-rgb'' keyword</h3>
```

Comment:
No, [opRGB](https://www.color.org/chardata/rgb/oprgb.xalter) is fairly similar,but not precisely the same.